python/pyspark/pandas/namespace.py

```diff
@@ -272,7 +272,7 @@ def read_csv(
         The character used to denote the start and end of a quoted item. Quoted items can include
         the delimiter and it will be ignored.
     escapechar : str (length 1), default None
-        One-character string used to escape delimiter
+        One-character string used for escaping quotes inside an already quoted value.
```

Comment:
I think it just follows the pandas documents, so maybe we can just keep it as is if there is no strong necessity.

Following the pandas as much as possible is the goal we aim for the most in pandas API on Spark.

Reply:
And maybe the `escaper` is not only for escaping quotes inside an already quoted values ??

I think It can be any 1-length string other than quotes?

Reply:
Good point @itholic - I have updated the PR to reflect the Pandas docs. 

yes, this param is used to escape more than just the quotes, but also more than just the delimiter char (as suggested by the current language).